Sources/TuistLoader/Mappers/ProjectDescriptionHelpersLocationMapper.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  ProjectDescriptionHelpersLocationMapper.swift
+//  TuistLoader
+//
+//  Created by Luis on 11/10/20.
+//
+
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+/// Mapper that changes the development region based on configuration
+/// in the Config.swift
+public final class ProjectDescriptionHelpersLocationMapper: ProjectMapping {
+    private let config: TuistCore.Config
+    private let rootDirectoryLocator: RootDirectoryLocating
+
+    private lazy var derivedHelpersDirectory: AbsolutePath? = {
```

Comment:
I think this can be turned into a function `private func derivedHelpersDirectory() throws -> AbsolutePath?`

That way you:
- leverage error handling that is already implemented up the chain
- do not repeat `logger.error` call

To reuse the error message, I'd create a new error that you can throw

This also ensures that the whole pipeline fails when the derived directory is not found -> that is something we want as otherwise we get into unknown state where we can not ensure the right results.

Besides that, I'd also create a test for this error being thrown.

Reply:
This makes sense, updating!